Compute/README.md

```diff
@@ -0,0 +1,57 @@
+# Google Compute for PHP
+
+> Idiomatic PHP client for [Google Compute](https://cloud.google.com/compute/).
+
+[![Latest Stable Version](https://poser.pugx.org/google/cloud-compute/v/stable)](https://packagist.org/packages/google/cloud-compute) [![Packagist](https://img.shields.io/packagist/dm/google/cloud-compute.svg)](https://packagist.org/packages/google/cloud-compute)
+
+* [API Documentation](http://googleapis.github.io/google-cloud-php/#/docs/cloud-compute/latest)
+
+**NOTE:** This repository is part of [Google Cloud PHP](https://github.com/googleapis/google-cloud-php). Any
+support requests, bug reports, or development contributions should be directed to
+that project.
+
+Allows you to create, manage, share and query data.
+
+### Installation
+
+To begin, install the preferred dependency manager for PHP, [Composer](https://getcomposer.org/).
+
+Now to install just this component:
+
+```sh
+$ composer require google/cloud-compute
+```
+
+Or to install the entire suite of components at once:
+
+```sh
+$ composer require google/cloud
+```
+
+### Authentication
+
+Please see our [Authentication guide](https://github.com/googleapis/google-cloud-php/blob/master/AUTHENTICATION.md) for more information
+on authenticating your client. Once authenticated, you'll be ready to start making requests.
+
+### Sample
+
+```php
+require 'vendor/autoload.php';
+
+use Google\Cloud\Compute\V1\InstancesClient;
+
+$instances = new InstancesClient();
+foreach ($instances->list_() as $instance) {
+    print($instance->getName());
+}
+```
+
+### Version
+
+This component is considered GA (generally available). As such, it will not introduce backwards-incompatible changes in
```

Comment:
I assume we'll want the beta message here :)

```
This component is considered beta. As such, it should be expected to be mostly stable and we're working towards a release candidate. We will address issues and requests with a higher priority.
```

Reply:
Oops! 
This component is actually considered alpha. Have we used alpha messaging before?

Reply:
We used to - but some time back there was unliteral decision to only have two broad versioning levels, beta/GA.

Reply:
We could use different language to avoid the term "alpha", such as "This component is in active development". What do you think?

Reply:
That works for me